docs/learn/learn-identity.md

```diff
@@ -152,6 +116,34 @@ for the `max_fee` put the maximum you're willing to pay for these confirmations.
 If you don't know which registrar to pick, first check the available registrars by going to
 ["Chain State UI"](#) and selecting `identity.registrars()` to get the full list.
 
+### Requesting Judgement
+
+Requesting judgement follows the same process regardless of whether you're on the Kusama or Polkadot
+networks. Select one of the registrars from the query you made above.
+
+![Requesting judgement](../assets/identity/08.jpg)
+
+This will make your identity go from unjudged:
+
+![An unjudged identity](../assets/identity/07.jpg)
+
+To "waiting":
+
+![A pending identity](../assets/identity/09.jpg)
+
+At this point, direct contact with the registrar is required - the contact info is in their identity
+as shown above. Each registrar will have their own set of procedures to verify your identity and
+values, and only once you've satisfied their requirements will the process continue.
+
+Once the registrar has confirmed the identity, a green checkmark should appear next to your account
+name with the appropriate confidence level:
+
+![A confirmed identity](../assets/identity/10.jpg)
+
+_Note that changing even a single field's value after you've been verified will un-verify your
+account and you will need to start the judgement process anew. However, you can still change fields
+while the judgement is going on - it's up to the registrar to keep an eye on the changes._
+
 ## Cancelling Judgements
```

Comment:
Should be `### Cancelling Judgements` (3x `#`), no?

Reply:
Thanks, updated.